wormhole_chain/app/cosmwasm_handlers.go

```diff
@@ -0,0 +1,213 @@
+package app
```

Comment:
minor nit: it may be worth putting all the different handlers in separate files in a separate package

Reply:
done